app/models/question.rb

```diff
@@ -1,6 +1,17 @@
 class Question < ActiveRecord::Base
 
   has_many :answers
-  belongs_to :user
+  belongs_to :user, inverse_of: :questions
+
+  def self.all_by_query_params(params)
+    questions = self.where(private: false)
+    if params[:question_name]
+      questions = questions.where('title LIKE ?', '%' + params[:question_name] + '%')
```

Comment:
@jerelmiller, I think you need the `%` in the parameter. When the value is interpolated, it is surrounded by quotes, which results in `"title LIKE \"%'my-title'%\""`